src/main/java/uk/gov/hmcts/reform/sscs/util/PdfRequestUtil.java

```diff
@@ -170,21 +178,45 @@ private static DocumentLink getPreviewDocument(
             .build();
     }
 
-    public static DocumentType getPostHearingDocumentType(SscsCaseData sscsCaseData) {
-        PostHearing postHearing = sscsCaseData.getPostHearing();
-        DocumentType documentType;
-
-        switch (postHearing.getRequestType()) {
-            case SET_ASIDE:
-                documentType = DocumentType.SET_ASIDE_APPLICATION;
-                break;
-            case CORRECTION:
-                documentType = DocumentType.CORRECTION_APPLICATION;
-                break;
-            default:
-                throw new IllegalArgumentException("Unexpected request type: " + postHearing.getRequestType());
+    public static DocumentType getPostHearingDocumentType(PostHearingRequestType postHearingRequestType) {
+        if (nonNull(postHearingRequestType)) {
+            switch (postHearingRequestType) {
+                case SET_ASIDE:
+                    return DocumentType.SET_ASIDE_APPLICATION;
+                case CORRECTION:
+                    return DocumentType.CORRECTION_APPLICATION;
+                case STATEMENT_OF_REASONS:
+                    return DocumentType.STATEMENT_OF_REASONS_APPLICATION;
+                default:
+                    break;
+            }
         }
-        return documentType;
+
+        throw new IllegalArgumentException("Unexpected request type: " + postHearingRequestType);
```

Comment:
As above, put the exception back to the default case

Reply:
done